bika/lims/content/abstractroutineanalysis.py

```diff
@@ -477,13 +478,19 @@ def getDependencies(self):
     @security.public
     def getPrioritySortkey(self):
         """
-        Returns the key that will be used to sort the current Analysis
-        Delegates to getPrioritySortKey function from the AnalysisRequest
+        Returns the key that will be used to sort the current Analysis, from
+        most prioritary to less prioritary.
         :return: string used for sorting
         """
         analysis_request = self.getRequest()
         if analysis_request:
-            return analysis_request.getPrioritySortkey()
+            ar_sort_key = analysis_request.getPrioritySortkey()
+            ar_id = analysis_request.getId().lower()
+            title = sortable_title(self)
+            if callable(title):
+                title = title()
+            return '{}.{}.{}'.format(ar_sort_key, ar_id, title)
```

Comment:
Just a little style issue here to remove code complexity.

Better do something like this:
```
analysis_request = self.getRequest()
if analysis_request is None:
    return None
# continue with the rest of the code with a lower indendation level
ar_sort_key = analysis_request.getPrioritySortkey()
...